manifests/init.pp

```diff
@@ -0,0 +1,41 @@
+# @summary
+#   Creates a xfs_quota bash script in /etc/telegraf/scripts, and a conf file to run it
+#
+# @param enable
+#   Enables the xfs quota checks
+# 
+# @param quota_path
+#   Specifies the path of the xfs volume
+#
+# @example
+#   include profile_xfs_quota
+class profile_xfs_quota (
+  Boolean $enable,
+  String $quota_path,
```

Comment:
My personal preference is to not repeat the word `quota` here, i.e. the parameter is currently `profile_xfs_quota::quota_path`. I'd suggest either just simply `profile_xfs_quota::path` or perhaps `profile_xfs_quota::volume_path`.

```suggestion
  String $path,
```

Again, not a big deal, but I think that would be cleaner.

Reply:
I like that suggestion, it makes more sense now that it part of it's own profile. I will go with volume_path for now.